draft-ietf-quic-qlog-h3-events.md

```diff
@@ -378,8 +378,23 @@ Owner = "local" / "remote"
 
 ## HTTPFrame
 
+The generic `$HTTPFrame` is defined here as a CDDL extension point (a "socket"
+or "plug"). This allows other documents to extend this extension point when
+defining new HTTP/3 protocol frame types to enable automated validation of
+aggregated qlog schemas.
```

Comment:
The intent of the sentence isn't that clear to me. Who is wanting to automate validation, does somebody producing qlog or consuming qlog care?

How about simplifying to :

```suggestion
or "plug"). It can be extended to support additional HTTP/3 frame types.
```